tests/gumball-machine-test.ts

```diff
@@ -27,10 +28,11 @@ import {
   decodeGumballMachine, OnChainGumballMachine
 } from "./gumball-machine-serde";
 import NodeWallet from "@project-serum/anchor/dist/cjs/nodewallet";
-import { getAssociatedTokenAddress } from "../deps/solana-program-library/token/js/src";
+import { getAssociatedTokenAddress, createMint, getOrCreateAssociatedTokenAccount, mintTo, getAccount } from "../deps/solana-program-library/token/js/src";
```

Comment:
using the js libs from the solana library this way usually causes the other anchor tests to break

I hope I'm wrong here, but do the other tests pass in `anchor test` when you have this?

Reply:
For me locally all tests were passing except for gummyroll continuous, which I think you fixed on another branch :)

Reply:
sg